src/main/java/seedu/ezmanager/commands/member/AssignMemberToProjectCommand.java

```diff
@@ -43,6 +51,18 @@ public void parse() throws EzExceptions {
         }
     }
 
+    /**
+     * Assign specified member to specified project.
```

Comment:
would writing it like this making it easier to understand?
```java
/**
     * Assign specified member to specified project. The sequence of steps executed are:
     * 1. Add TeamMember object to ArrayList of TeamMember in specified Project.
     * 2. Add Project object to ArrayList of Project in specified TeamMember.
     * 3. Prints member assigned to project message.
     *
     * @param projects ArrayList of Projects.
     * @param teamMembers ArrayList of TeamMembers in the program.
     * @return Print member assigned to project message.
     * @throws EzExceptions Invalid TeamMember ID if when parameter values provided is outside
     *     the range of the TeamMember list.
     */
```

Reply:
done!!